README.md

```diff
@@ -243,7 +243,7 @@ Codec 2 requires a hardware Floating Point Unit (FPU) to run in real time.
 Two build options have been added to support building on microcontrollers:
 1. Setting the `cmake` variable MICROCONTROLLER_BUILD disables position independent code (-fPIC is not used).  This was required for the IMRT1052 used in Teensy 4/4.1).
```

Comment:
Actually, removing -fPIC was required for every microcontroller I tried (Teensy, Pi Pico and ESP32). I suspect it's required for most if not all of them but then again, the SM1000 doesn't need this option so 🤷